tox.ini

```diff
@@ -23,3 +23,15 @@ basepython = python3.7
 
 [testenv:py38]
 basepython = python3.8
+
+
+[testenv:checks]
+basepython = python3.8
+deps =
+    black>=19.10b0
+    flake8>=3.7.9
+    mypy>=0.770
```

Comment:
I'd make mypy a separate environment, and pin mypy version. Maybe things changed, but my experience (for a much larger codebase though) mypy update is quite likely to start indicating new issues.